aikau/src/main/resources/alfresco/editors/TinyMCE.js

```diff
@@ -226,14 +226,23 @@ define(["dojo/_base/declare",
          // Check that the language requested is supported...
          if (config.language) {
             var locales = this.supportedLocales.split(",");
-            var locale = "en";
+            var locale, bestGeneralizedLocale;
             for (var i = 0, j = locales.length; i < j; i++) {
                if (locales[i] === config.language) {
                   locale = config.language;
                   break;
                }
+               
+               if (config.language.indexOf(locales[i]) === 0)
+               {
+                   if (bestGeneralizedLocale === undefined || locales[i].length > bestGeneralizedLocale.length)
+                   {
+                       bestGeneralizedLocale = locales[i];
+                   }
+               }
             }
-            config.language = locale;
+            
+            config.language = locale || bestGeneralizedLocale || 'en';
```

Comment:
We have JSHint on our build servers to use double rather than single quotes (the build aggressively fails on JSHint errors!). I'm flagging this up for future reference more than anything else as I'm happy to merge myself and then fix the build failure. But if it's simple enough for you to push a quick update then I'd be very grateful.